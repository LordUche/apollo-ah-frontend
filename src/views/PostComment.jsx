
import React from 'react';
import { PropTypes } from 'prop-types';

export default function PostComment({ postComment }) {
  return (
    <form className="form" onSubmit={postComment}>
      <div className="form-group">
        <input className="form-control" placeholder="Leave a comment" name="body" />
        <button type="submit" className="btn btn-link">
          <i className="fas fa-paper-plane" />
        </button>
      </div>
    </form>

  );
}

PostComment.propTypes = {
  postComment: PropTypes.func.isRequired,
};
