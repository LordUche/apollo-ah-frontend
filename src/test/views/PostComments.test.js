import React from 'react';
import { shallow } from 'enzyme';
import PostComment from '../../views/PostComment';


describe('<PostComment>', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<PostComment postComment={jest.fn()} />);
    expect(wrapper).toMatchSnapshot();
  });
});
