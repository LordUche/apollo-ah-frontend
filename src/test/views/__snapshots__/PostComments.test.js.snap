// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostComment> should render without crashing 1`] = `
<form
  className="form"
  onSubmit={[MockFunction]}
>
  <div
    className="form-group"
  >
    <input
      className="form-control"
      name="body"
      placeholder="Leave a comment"
    />
    <button
      className="btn btn-link"
      type="submit"
    >
      <i
        className="fas fa-paper-plane"
      />
    </button>
  </div>
</form>
`;
